xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/config/XmDomainEventConfiguration.java

```diff
@@ -0,0 +1,103 @@
+package com.icthh.xm.commons.domain.event.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.icthh.xm.commons.config.client.api.RefreshableConfiguration;
+import com.icthh.xm.commons.tenant.TenantContextHolder;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.util.AntPathMatcher;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@Component
+public class XmDomainEventConfiguration implements RefreshableConfiguration {
+
+    private static final String TENANT_NAME = "tenant";
+
+    private final AntPathMatcher matcher = new AntPathMatcher();
+    private final ObjectMapper ymlMapper = new ObjectMapper(new YAMLFactory());
+    private final TenantContextHolder tenantContextHolder;
+    private final LiquibaseRunner liquibaseRunner;
+    private final String configPath;
+
+    private final Map<String, EventPublisherConfig> configByTenant = new HashMap<>();
+
+
+    public XmDomainEventConfiguration(@Value("${spring.application.name}") String appName,
+                                      TenantContextHolder tenantContextHolder,
+                                      LiquibaseRunner liquibaseRunner) {
+        this.tenantContextHolder = tenantContextHolder;
+        this.configPath = "/config/tenants/{tenant}/" + appName + "/domain-events.yml";
+        this.liquibaseRunner = liquibaseRunner;
+    }
+
+    public SourceConfig getInterceptorConfig(String source) {
```

Comment:
here method should be named `getSourceConfig()`
I've forgot to rename it on the diagram when changes Interceptor to Source abstraction.